lib/envlogic/env.rb

```diff
@@ -28,9 +32,22 @@ def initialize(klass)
     # @example
     #   env.update('production')
     def update(environment)
-      replace(
-        ActiveSupport::StringInquirer.new(environment)
-      )
+      replace(environment)
```

Comment:
maybe make alias if it just calls another method?

Reply:
that is indeed a good idea!